src/services/viewport-listerner/viewport-listener.module.ts

```diff
@@ -0,0 +1,9 @@
+import { NgModule } from '@angular/core';
+import { ViewportListenerService } from './viewport-listener.service';
+import { ResizeModule } from '../../../dist/directives/resize';
```

Comment:
I think this is the problem, you've accidentally imported from the dist directory.

Reply:
Thanks!